### PR TITLE
modified function for heavy atoms

### PR DESCRIPTION
**Pull request by JohannesKarwou**
## Description
Its now possible to specify the number of the heavy atoms which should be turned off within the original yaml file

## Status
- [x] Needs to be tested

### CODE REVIEW

File: transformato/schrodinger_systems.py [hunk 1]
```diff
@@ -7,7 +7,144 @@
...
+    )
+
+    # write endpoint mutation
+    output_file_base, intst = i.write_state(mutation_conf=[], intst_nr=1)
+    output_files.append(output_file_base)
+
+    # start with charges
+    # turn off charges
+    charges = mutation_list["charge"]
+    nr_of_mutation_steps_charge = (
```
Comment (wiederm) [inline transformato/schrodinger_systems.py:45]: can you also put the number of mutation steps for the different mutations in the yaml file?

**Review by wiederm — COMMENTED** [review]
This looks good to me!
One minor comment and suggestion:
I'd add a new top level tag and use it instead of `simulation`, e.g. `mutation`. I'd put everything in here that can be user defined, so I'd put also values for the number of mutation steps for the specific mutations.

File: transformato/schrodinger_systems.py (hunk 1 above)
Comment (JohannesKarwou) [reply transformato/schrodinger_systems.py]: Good idea! I added them for each structure...

**Review by wiederm — APPROVED** [review]
looks great!

**Comment by JohannesKarwou** [comment]
To Do
[ ] Postprocessing needs to be adapted
[ ] Change of psf file from `0    0   !NGRP NST2` to    `1    0 !NGRP NST2 ` causes trouble in charmm_openmm

**Pull request merged**